streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Resource.java

```diff
@@ -82,4 +82,29 @@ public class Resource implements Serializable {
   private transient String sortOrder;
 
   private transient String connector;
+
+  public void setResourcePath(String resourcePath) {
+    if (resourcePath == null) {
+      throw new IllegalArgumentException("resource path cannot be null.");
+    }
+    String[] namePath = resourcePath.split(":");
+    if (namePath.length != 2) {
```

Comment:
@ChengJie1053 Thx for the contribution~
Could we use Utils.require to replace lines `87`, `91` ?

Reply:
Ok, thanks for reviewing the code